components/Form/Dropdown.tsx

```diff
@@ -0,0 +1,153 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
What is this dropdown form for? I believe we already have a dropdown component. OK to add another, just need to understand the use case

Reply:
Right. We have `GenericMenu` which is just a generic dropdown used in a bunch of places. We also have a few others I believe but that one is pretty light weight

Reply:
Ah yeah this was leftover from the branch with RSC — removed the component.